src/bundle/Resources/views/themes/admin/ui/component/input_date_and_time.html.twig

```diff
@@ -0,0 +1,36 @@
+{% import '@ibexadesign/ui/component/macros.html.twig' as html %}
+
+{% set is_small = is_small|default(false) %}
+{% set inline_flatpickr = inline_flatpickr|default(false) %}
+
+{% set wrapper_attr = wrapper_attr|default({})|merge({
+    class: (wrapper_attr.class|default('')
+        ~ ' ibexa-date-time-picker'
+        ~ (is_small ? ' ibexa-date-time-picker--small')
+        ~ (inline_flatpickr ? ' ibexa-date-time-picker--inline-flatpickr')
+    )|trim
+}) %}
+{% set input_attr = input_attr|default({})|merge({
+    class: (input_attr.class|default('')
+        ~ ' ibexa-input ibexa-input--text form-control ibexa-date-time-picker__input flatpickr flatpickr-input'
+        ~ (is_small ? ' ibexa-input--small')
+    )|trim,
+    readonly: 'readonly',
+    type: 'text',
+}) %}
+
+{% set input_attr_rendered = html.attributes(input_attr) %}
+
+{% embed '@ibexadesign/ui/component/input_text.html.twig' with { has_search: false } %}
+    {% block content %}
+        <input {{ input_attr_rendered }} />
```

Comment:
nitpick: I think this is simpler and clearer.
```suggestion
        <input {{ html.attributes(input_attr) }} />
```

Reply:
But it's not working, it claims that html is not defined

Reply:
You have to import `html` inside `embed`.